README.md

```diff
@@ -66,11 +68,13 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [Selenide](https://github.com/codeborne/selenide) - A framework for writing easy-to-read and easy-to-maintain automated tests using a Fluent API. Selenide has a magic trick that resolves most problems with Ajax and timeouts.
 - [Galen Framework](http://galenframework.com/) - Automated testing of look and feel for your responsive websites.
 - [Serenity](http://www.thucydides.info/) - It is an open source library for writing better quality automated acceptance tests faster. (Formerly Thucydides).
+- [SeLion](https://github.com/paypal/SeLion) - A framework for running Selenium tests with additional features beyond the basic Selenium Grid functionality.
```

Comment:
You already defined this under Selenium Grid

Reply:
Again, let me guess, no duplicate listings whatsoever? If that's the case, be interesting to decide where an item best fits when it spans multiple categories.

Reply:
same here